sound/soc/sof/intel/hda.h

```diff
@@ -346,6 +346,10 @@
 /* Host Device Memory Size of a Single SSP */
 #define SSP_DEV_MEM_SIZE	0x1000
 
+/* SSP Count of the Platform */
+#define APL_SSP_COUNT		6
+#define CNL_SSP_COUNT		8
```

Comment:
no, I only see 3 SSPs for CNL/WHL/CML. You are confusing IP options and SOC reality, please look at the right documentation.

Reply:
OK, changed.